src/dashboard/cd-dashboard.vue

```
@@ -6,7 +6,7 @@
         <dashboard-projects/>
         <dashboard-news />
       </div>
-      <div class="cd-dashboard__right-column">
+      <div class="cd-dashboard__right-column" v-if="userProfile && (childrenAreVisible || statsAreVisible)">
```

Show by default, hide when we know there's nothing to show